sync.py

```diff
@@ -22,6 +22,11 @@ def simple(input_string):
     # only take the first part of a string before any hyphens or brackets to account for different versions
     return input_string.split('-')[0].strip().split('(')[0].strip().split('[')[0].strip()
 
+def isrc_match(tidal_track, spotify_track):
+    if "isrc" in spotify_track["external_ids"]:
```

Comment:
Do we need to check that isrc is not an empty string?

Reply:
Yes, due to some tracks having None fields, e.g. the id and in this case the isrc external id. I got a KeyError lookup exception when running the new function against my library.

Reply:
I'm not following you... If we need to check that it's not empty string than can you please add this?

Reply:
Sorry, misunderstood. Thought you were asking about the current if statement, my bad.
Given what I tested and seen so far, I do not think it is needed. I mean, it doesn't hurt, but since the ISRC is mandatory in the music industry, if the key for it is available in the dict, it most likely is there and in good shape.